tdrs-backend/tdpservice/settings/cloudgov.py

```diff
@@ -126,17 +126,22 @@ class Development(CloudGov):
 
     # https://docs.djangoproject.com/en/2.0/ref/settings/#allowed-hosts
     ALLOWED_HOSTS = ['.app.cloud.gov']
+    AV_SCAN_URL = os.getenv('AV_SCAN_URL', 'http://tanf-dev-clamav-rest.apps.internal:9000')
+    MIDDLEWARE = ('django.contrib.sessions.middleware.SessionMiddleware', *Common.MIDDLEWARE)
```

Comment:
I am unsure if the above two lines are needed. They might belong in CloudGov or Common.